README.md

```diff
@@ -158,7 +158,7 @@ See also
 [testing]: https://www.openpolicyagent.org/docs/latest/policy-testing/
 [docs]: https://hacbs-contract.github.io/
 [hacbsdocs]: https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/ec-policies/index.html
-[taskdef]: https://github.com/redhat-appstudio/build-definitions/blob/main/tasks/verify-enterprise-contract.yaml
+[taskdef]: https://github.com/hacbs-contract/ec-cli/blob/main/task/0.1/verify-enterprise-contract.yaml
```

Comment:
should this just link to the file within this repo? I think this will work:
```
[taskdef]: task/0.1/verify-enterprise-contract.yaml
```

Reply:
nevermind. that's a different repo.

Reply:
You mean ec-policy repo can refer task definition in ec-cli this way?

Reply:
I thought this was a PR on the ec-policies repo :grimacing: 
It is not, so just ignore my comment